### DISCUSSION TITLE
unable to connect to channel deployed in mirth docker image

### DISCUSSION THREAD

**Discussion by mkotaprolu** [nextgenhealthcare/connect#6078] [Q&A] [answered]
Hi Team,

I was able to deploy channel to docker mirth container locally, but i am unable to send messages to rest api listener source connector for the channel deployed, from outside the docker container from post man or CURL locally.

deployed channel rest api listener port configured: 9000

<img width="1685" alt="image" src="https://github.com/nextgenhealthcare/connect/assets/80752550/e4437324-9a36-4c99-959d-4cca5cb5eac5">

channel deployed is verified by mccommand into docker mirth server  from another installation of mirth on same machine

./mccommand -a https://localhost:55000 -u admin -p admin

<img width="1627" alt="image" src="https://github.com/nextgenhealthcare/connect/assets/80752550/2f72f1b0-b532-4684-a556-6769471dd5e8">

please let me know how to send the messages in the channel deployed using post man or CURL locally.

Docker image: https://hub.docker.com/r/nextgenhealthcare/connect
Os : Mac os ventura 13.5

**Comment by pacmano1** [comment, marked as answer]
That is a docker question, not a mirth question.  You need to expose container ports.

**Comment by mkotaprolu** [comment]
expose docker ports for mirth server and channel level port while starting the container
